include/sprite.h

```diff
@@ -19,6 +19,16 @@
 #define SpriteGetScreenPos(spritePos, regionPos)                                        \
     ((spritePos)*TILE_WIDTH + (regionPos)*CAM_REGION_WIDTH)
 
+#define CAM_BOUND_X ((DISPLAY_WIDTH) + (CAM_REGION_WIDTH))
```

Comment:
As discussed, probably move to `game.h` for now and we'll find a proper place later. Same with `SpriteGetScreenPos` and others relating to Entities now thinking about it